docs/install/lotus/README.md

```diff
@@ -0,0 +1,137 @@
+
+The Lotus client can be used for one of three use-cases:
+
+| Client        | Purpose                                                                                                                                                                         |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Lotus node    | An interface between your computer and the Filecoin network. All transactions, queries, and interactions with the Filecoin network must pass through a Lotus node.              |
+| Storage miner | The Lotus storage miner queries a Lotus node for the latest tipset on-chain, produces a block on that tipset, and submits it to the node to be broadcast to the entire network. |
+| Seal worker   | An extra process that can offload heavy processing tasks from your Lotus Storage miner.                                                                                         |
+
+All three of these use-cases use the same Lotus client, but they differ in how they are configured _after_ the client has been installed. To make the install go as smoothly as possible, read through the [software and hardware requirements](./requirements) first.
+
+## Linux
+
+1.  Install the dependencies:
+
+    ```bash
+    sudo apt update
+    sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config curl build-essential -y
+    ```
+
+1.  Clone the Lotus repository:
+
+    ```bash
+    git clone https://github.com/filecoin-project/lotus.git
+    ```
+
+1.  _Build_ and _Make_ the Lotus project:
+
+    ```bash
+    make clean && make all
+    sudo make install
+    ```
+
+    This process might take a while
+
+1.  Once the build process has finished, you're left with a `lotus` executable:
+
+    ```bash
+    ./lotus --version
+
+    > lotus version 0.4.1+git.7d7496e1
+    ```
+
+That's it! You've now got Lotus installed on your system. If you'd like to be able to run `lotus` from anywhere on your system:
+
+1. Move the `lotus` executable to `/usr/local`:
+
+   ```bash
+   sudo mv lotus /usr/local/bin
+   ```
+
+1. Add `lotus` to your system PATH:
+
+   ```bash
+   echo "export PATH=\$PATH:/usr/local/bin/lotus" >> ~/.profile
+   source ~/.profile
+   ```
+
+1. You can now run `lotus` commands from anywhere:
+
+   ```bash
+   cd ~
+   lotus --version
+
+   > lotus version 0.4.1+git.7d7496e1
+   ```
+
+## macOS
+
+1. Make sure you have `xcode-select` and Homebrew installed:
+
+   ```bash
+   xcode-select --install
+
+   brew --version
+   ```
+
+1. Install the dependencies for Lotus:
+
+   ```bash
+   brew install go bzr jq pkg-config rustup git
```

Comment:
Needs a remark saying Go 1.15 doesn't work

Reply:
Correct. The [Requirements page](https://deploy-preview-305--filecoin-docs.netlify.app/install/lotus/requirements/#software) lists that. Again, in the interest of getting this actually merged in, I'm going to resolve this conversation. Please create an issue to fix your comment, or submit a PR to fix it.